src/algorithms/calorimetry/CalorimeterClusterRecoCoG.h

```diff
@@ -13,6 +13,8 @@
 
 #include <random>
 
+#include <Eigen/Dense>
```

Comment:
This header is an implementation detail, should go to the .cc file.

Reply:
Done.